core/controllers/profile.py

```diff
@@ -77,6 +77,12 @@ def get(self, username):
         if not user_settings:
             raise self.PageNotFoundException
 
+        if user_settings.first_contribution_datetime:
```

Comment:
I chose to do the original if statement because 'utils.get_time_in_millisecs' crashed when first_contribution_datetime was none. I see the point in your new code, however! I will modify my code to take in the new format.